Doc/library/http.client.rst

```diff
@@ -395,6 +401,13 @@ also send your request step by step, by using the four functions below.
    called.
 
 
+.. attribute:: HTTPConnection.blocksize
```

Comment:
This is in a list of “four functions” for low-level requests. Perhaps move it up into the main list, and adjust or remove “the following methods”.

Reply:
Ok, will move up before the low level method section.